src/main/java/org/apache/pirk/querier/wideskies/QuerierCLI.java

```diff
@@ -131,12 +125,28 @@ private boolean parseOptions()
     // Validate properties
     valid = QuerierProps.validateQuerierProperties();
 
+    // Load the new local query and data schemas
+    if (valid)
+    {
+      logger.info("loading schemas: dataSchemas = " + SystemConfiguration.getProperty("data.schemas") + " querySchemas = " + SystemConfiguration
+          .getProperty("query.schemas"));
+      try
+      {
+        DataSchemaLoader.initialize();
+        QuerySchemaLoader.initialize();
+
+      } catch (Exception e)
```

Comment:
It would be much better to handle the exception, or re-throw it, rather than consume and continue.
Why catch all Exception types?